bitcoin/src/blockdata/witness.rs

```diff
@@ -323,6 +323,23 @@ impl Witness {
         self.push(&sig[..signature.len() + 1]);
     }
 
+    /// Pushes a segwit v0 signature on the witness, requires an allocation.
+    pub fn push_segwit_v0_signature(&mut self, sig: &SegwitV0Signature) {
```

Comment:
Note to self, this is wrong, its only for `p2wpkh` not `p2wsh` so the name is wrong.